src/introspective.jl

```diff
@@ -3,41 +3,45 @@ CACHETIMEOUT = 30 # s
 
 MAX_RETURN_SIZE = 20 # how many results to return at most
 
-
-function searchdocs(needle::String; loaded = true, mod = "Main",
-                    maxreturns = MAX_RETURN_SIZE, exportedonly = false,
-                    name_only = false)
+function searchdocs(needle::AbstractString; loaded::Bool = true, mod::Module = Main,
```

Comment:
We can keep `mod` to be `String` if we have something like `Atom.getmodule` in DocSeeker. It's would be better when we search docs in unloaded modules (currently the passed `mod == Main` for this case, and so the new [module unmatch penalty](https://github.com/JunoLab/DocSeeker.jl/blob/267f49b8ba93a51142dd9067e1c988c4091aaa66/src/DocSeeker.jl#L66-L67) doesn't work.

What do you think on this ?